src/general/OptionsMenu.tscn

```diff
@@ -2011,7 +2011,6 @@ margin_top = 360.0
 margin_right = 940.0
 margin_bottom = 360.0
 rect_min_size = Vector2( 600, 0 )
-popup_exclusive = true
```

Comment:
Shouldn't this at least be exclusive as the player *has* to do something to resolve the going back problem before they can do anything else?